readme.md

```diff
@@ -51,6 +51,10 @@ unicorn('yo');
 
 Test if `value` matches the provided `predicate`.
```

Comment:
It's now unclear what this does. Can you add something about it throwing an error message?